CHANGELOG.md

```diff
@@ -1,370 +1,35 @@
-- Allowing to disable local-by-default for transactions with new configurations ([#8882](https://github.com/paritytech/parity-ethereum/pull/8882)).
-- Never drop local transactions from different senders ([#9002](https://github.com/paritytech/parity-ethereum/pull/9002)).
-- Optimize pending transactions filter and fix ethstats reporting of pending transactions ([#9026](https://github.com/paritytech/parity-ethereum/pull/9026)).
-- Add separate database directory for light client allowing to run full and light nodes at the same time ([#9064](https://github.com/paritytech/parity-ethereum/pull/9064)).
+- This release moves the 2.1 track of Parity-Ethereum to beta.
+- This release contains a low-severity issue with the web-sockets ports. [#9545](https://github.com/paritytech/parity-ethereum/pull/9545)
+- This release resolves a potential network fragmentation issue. [#9526](https://github.com/paritytech/parity-ethereum/pull/9526)
+- With this release, all versions of Parity Ethereum 1.x prior to 2.0 reached end of life.
+- Users are urged to upgrade to 2.0.5-stable or 2.1.0-beta.
 
-If you are upgrading directly from versions 1.10.9 or earlier, please note important changes to our transaction-queue implementation, namely:
+Further changes worth highlighting:
 
-- The pool now limits transactions per-sender (see `--tx-queue-per-sender`), local transactions also have to obey that limit. Consider increasing the limit via CLI-flag when running benchmarks or sending a lot of transactions at once.
-- In case the pool is full, transactions received over the network, but originating from accounts that you have private keys for might not get accepted to the pool any more with higher priority. Consider running with larger pool size or submitting the transactions directly on the node via `eth_sendRawTransaction`.
+- Generalized `blockReward` and `difficultyBombDelays` config ([#9480](https://github.com/paritytech/parity-ethereum/pull/9480)): This removes `eip649*` parameters from chain specs and allows `blockReward` to accept `multi`. Please review your chain-specs!
+- Implement EIP234 `block_hash` for `eth_getLogs` ([#9256](https://github.com/paritytech/parity-ethereum/pull/9256)): If `block_hash` and `from_block`/`to_block` present, return error. This also changes `eth_getLogs` to return error if any of `block_hash`/`from_block`/`to_block` cannot be found.
+- The default `gas_floor_target` was increased to `8_000_000`, the default `gas_cap` to `10_000_000`.
+- Comply EIP-86 with the new definition ([#9140](https://github.com/paritytech/parity-ethereum/pull/9140)). Note that this is backward incompatible, with the assumption that none (or not many) working chains have attempted to set the `eip86Transition` flag, because it's not really useful by itself.
```

Comment:
EIP-86 flag has been removed (and replaced by EIP-1014) in #9268. And looks like we did include it in v2.1.

Reply:
#9268 says "Add EIP-1014 transition config flag" - how could I know?

Please save everyone some time by adding proper labels and pull request descriptions. I can't look at the code diff of 150 PRs when compiling the changelog.

Reply:
Apparently that PRs' scope was changed because of some issues -- there was indeed a description: https://github.com/paritytech/parity-ethereum/pull/9268#issuecomment-412434759

But yeah I agree this can be done better, and I should have added releasenote tag / put a reference to that comment in issue description.